README.md

```diff
@@ -1,12 +1,10 @@
 # ark_automate
 
 [![GitHub stars](https://img.shields.io/github/stars/bptlab/ark_automate)](https://github.com/bptlab/ark_automate)
-![GitHub repo size](https://img.shields.io/github/repo-size/bptlab/ark_automate)
 [![GitHub open issues](https://img.shields.io/github/issues/bptlab/ark_automate)](https://github.com/bptlab/ark_automate/issues)
 [![GitHub open pull requests](https://img.shields.io/github/issues-closed/bptlab/ark_automate)](https://github.com/bptlab/ark_automate/issues)
 [![GitHub open pull requests](https://img.shields.io/github/issues-pr/bptlab/ark_automate)](https://github.com/bptlab/ark_automate/issues)
-
-<!-- [![heroku](heroku-badge.herokuapp.com/?app=ark-automate)](ark-automate.herokuapp.com) -->
+[![heroku](https://heroku-badge.herokuapp.com/?app=ark-automate&root=App.js)](https://heroku-badge.herokuapp.com/App.js)
 
 An open source RPA tool which uses BPMN to create bots.
```

Comment:
I would rather describe it as "An open source RPA tool which uses several modeling languages to create bots."

Reply:
I have adopted your text. Can you resolve the conversation? @WolfgangDaniel